src/mlpack/methods/neuro_cmaes/random.hpp

```diff
@@ -0,0 +1,88 @@
+#pragma once
```

Comment:
Do we need the random class? I think, even if we do we could use armadillo or mlpack core functions, maybe I missed something?